care/apps/patients/models.py

```diff
@@ -390,6 +390,13 @@ class PatientSampleTest(SoftDeleteTimeStampedModel):
     fast_track = models.TextField(default="")
     date_of_sample = models.DateTimeField(null=True, blank=True)
     date_of_result = models.DateTimeField(null=True, blank=True)
+    testing_lab = models.ForeignKey(
+        TestingLab,
+        on_delete=models.PROTECT,
+        related_name="testing_lab",
```

Comment:
it should be 'samples'

related_name would be used with calling from TestingLab instance so calling should be something like lab.samples